### PR TITLE
Added remark on that Outbox works in combination with MSMQ Distributor

### PR DESCRIPTION
**Pull request by ramonsmits**
Based on support call.

### CODE REVIEW

File: transports/msmq/distributor/configuration.md [hunk 1]
```diff
@@ -75,6 +75,9 @@ snippet: ConfiguringWorker
 
 Similar to self-hosting, if running NServiceBus prior to Version 6, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
 
+## Outbox
+
+The distributor can be used in combination with the [outbox](nservicebus/outbox/) feature. Outbox must be enabled on the Workers, and also on the Distributor node if configured as a Master. Outbox is only required for persistence operations, the forwarding of messages to workers is purely a transport operation and Outbox will not participate in the forwarding logic.
```
Comment (dvdstelt) [inline transports/msmq/distributor/configuration.md]: > and also on the Distributor node if configured as a Master

This is a different situation than when a node only behaves as distributor. I'd create a separate sentence of this, something like

`Outbox must be enabled on the Workers. It should also be enabled if a Worker is configured as Distributor Master.`

**Comment by ramonsmits** [comment]
@dvdstelt Applied feedback

**Comment by dvdstelt** [comment]
@Particular/docs-maintainers This can be merged. Should you do that?

**Comment by SzymonPobiega** [comment]
Yeah, you can self-merge when it build. The problem is related to the outbox page link.

**Comment by ramonsmits** [comment]
Rebased, hopefully resolved build issue.

**Comment by ramonsmits** [comment]
Rebased again...

**Pull request merged**